ios/MaasCore/Sources/MaasCore/Api/ApiConfiguration.swift

```diff
@@ -17,7 +17,13 @@ class ApiConfiguration: ApiConfig {
 
 public protocol ApiConfig {
     var baseUrl: String { get }
+    /**
+     - Tag: ApiConfig.apiKey
```

Comment:
What is the purpose of those tags? better generated documentation?

Reply:
So we can link from other source documentation using `x-source-tag://`. See e.g. the inline documentation for `ApiError.forbidden`.